pytket/binders/placement.cpp

```diff
@@ -136,7 +144,22 @@ PYBIND11_MODULE(placement, m) {
               const Architecture &, unsigned, unsigned, unsigned, unsigned>(),
           "The constructor for a GraphPlacement object. The Architecture "
           "object describes the connectivity "
-          "between qubits.\n\n:param arc: An Architecture object.\n"
+          "between qubits. To find a qubit to node assignment, this method "
+          "constructs a pattern graph where vertices are Circuit qubits and "
+          "edges mean a pair of qubits have an interaction in the circuit, "
+          "and then tries to find a weighted subgraph monomorphsim to the "
+          "architecture connectivity, or target, graph. Edges in the pattern "
+          "graph are weighted by the circuit depth the interaction between a "
+          "pair of qubit occurs at. "
```

Comment:
```suggestion
          "graph are weighted by the circuit depth at which the interaction between a "
          "pair of qubit occurs. "
```
Slightly easier to parse I think.